bookstore/_version.py

```diff
@@ -51,8 +51,8 @@ class NotThisMethod(Exception):
     """Exception raised if a method is not valid for the current scenario."""
 
 
-LONG_VERSION_PY: dict = {}
-HANDLERS: dict = {}
+LONG_VERSION_PY = {}
+HANDLERS = {}
```

Comment:
I like that you added this file to the exceptions list and then removed my half-hearted attempt to semi type this.